go.mod

```diff
@@ -1,25 +1,28 @@
 module github.com/brancz/kube-rbac-proxy
 
-go 1.23
+go 1.23.0
+
+toolchain go1.23.6
```

Comment:
I think we may not want to set this

Reply:
Yes, this was automagically set. I think it is fine as it indicates the minimal viable Go version in the sense that it might use features from v1.23.0+